### DISCUSSION TITLE
Add Relational Permissions (dot notation, nested)

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3144] [Feature Requests]
Allows you to specify access to certain fields when you're "coming in" through a parent item.

This allows you to return certain fields from a collection you don't have regular permissions too. For example, this would allow you to return the dimensions of a file, without giving read access to all directus_files

**Comment by Nitwel** [comment]
Any updates?

**Reply by hanneskuettner** [reply]
+1

**Reply by benhaynes** [reply]
++

**Reply by paescuj** [reply]
<3<3
